xml/System.Windows.Forms/Control.xml

```diff
@@ -4609,9 +4610,12 @@ The following table lists Windows Forms controls and which event (<xref:System.W
         <param name="dragImage">To be added.</param>
         <param name="cursorOffset">To be added.</param>
         <param name="useDefaultDragImage">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Begins a drag operation. The <paramref name="allowedEffects" /> determine which drag operations can occur. If the drag operation needs to interop with applications in another process, <paramref name="data" /> should either be a base managed class (<see cref="T:System.String" />, <see cref="T:System.Drawing.Bitmap" />, or <see cref="T:System.Drawing.Imaging.Metafile" />) or some <see cref="T:System.Object" /> that implements <see cref="T:System.Runtime.Serialization.ISerializable" />. <paramref name="data" /> can also be any <see cref="T:System.Object" /> that implements <see cref="T:System.Windows.Forms.IDataObject" />. <paramref name="dragImage" /> is the bitmap that will be displayed during the  drag operation and <paramref name="cursorOffset" /> specifies the location of the cursor within <paramref name="dragImage" />, which is an offset from the upper-left corner. Specify <see langword="true" /> for <paramref name="useDefaultDragImage" /> to use a layered window drag image with a size of 96x96; otherwise <see langword="false" />. Note the outer edges of <paramref name="dragImage" /> are blended out if the image width or height exceeds 300 pixels.</summary>
```

Comment:
Most of this needs to go in the remarks.

Reply:
Yeah, that would be a huge intellisense window 😆 .
```suggestion
        <summary>Begins a drag operation.</summary>
```